docs/ff-integrations/authentication/firebase-auth/google-login.md

```diff
@@ -37,21 +36,17 @@ You can create your own or use the one from page templates.
 
 ### Add Login Action
 
-- On your Google Login button, select **Actions** from the properties panel (the
-  right menu) and select Add Action.
-- Search and select the Log In (under **Backend/Database > Firebase
+1. On your Google Login button, select **Actions** from the properties panel (the
+  right menu) and select **Add Action**.
+2. Search and select the Log In (under **Backend/Database > Firebase
   Authentication**) action.
-- Set Auth Provider to Google.
+3. Set **Auth Provider** to **Google**.
+4. Enable **Create User Document** and set the **Collection** to **users**. After successful login, this will insert the user's details, such as email, name, and photo, into the *users* collection. If you haven’t already, see how to [create *users* collection](auth-initial-setup.md#creating-the-users-collection). **Note** that, If a user already exists, it won't add details again.
```

Comment:
```suggestion
4. Enable **Create User Document** and set the **Collection** to **users**. After successful login, this will insert the user's details, such as email, name, and photo, into the *users* collection. **Note** that, if a user exists already, it won't add details again.

If you haven’t already, see how to [create *users* collection](auth-initial-setup.md#creating-the-users-collection).
```

Reply:
Done.